### ISSUE TITLE
Audio issues when using videos with different frame rates

### ISSUE THREAD

**Issue by natasha-93** [JonnyBurger/remotion#320]
When using different videos, the audio doesn't fade out as expected. When testing, I found that it faded out some 400 frames after it should have. This could be related to the videos having different frame rates?

When using the same videos, the audio works as expected.

```
<>
  <Audio
    src="https://firebasestorage.googleapis.com/v0/b/ice-phoenix-6072f.appspot.com/o/music%2FCity%20Sunshine.mp3?alt=media"
    volume={interpolate(
    currentFrame,
    [durationInFrames - 50, durationInFrames],
    [1, 0],
      {
        extrapolateLeft: 'clamp',
        extrapolateRight: 'clamp',
      }
    )}
  />
  <div style={{zIndex: 5}}>
    <h1>{currentFrame}</h1>
  </div>
  <Sequence from={0} durationInFrames={150}>
    <Video src="https://firebasestorage.googleapis.com/v0/b/ice-phoenix-6072f.appspot.com/o/agencies%2FexampleAgency%2Fagents%2F5kLZBgkEfcguiGsc53kA%2Finput%2Fvideo%2Foutro-0.mp4?alt=media" />
  </Sequence>
  <Sequence from={150} durationInFrames={200}>
    <Video src="https://firebasestorage.googleapis.com/v0/b/ice-phoenix-6072f.appspot.com/o/agencies%2FexampleAgency%2Fagents%2F5kLZBgkEfcguiGsc53kA%2Fprojects%2F7KFKcmB0flUf2t5go3T6%2Fuploads%2Fvideo%2Fmain-1.mp4?alt=media" />
  </Sequence>
</>
```

**Comment by JonnyBurger** [comment]
Thanks for the report!
I created a repo and indeed something the audio doesn't fade out for me. I'll investigate and provide a bugfix hopefully soon!

**Comment by JonnyBurger** [comment]
Fixed in v2.0.7!
